modules/oe_translation_epoetry/src/MockAuthentication.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types = 1);
```

Comment:
Created our own because one in the library was in the test namespace, not available on sites.